packages/docs/docs/cli.md

```diff
@@ -15,12 +15,21 @@ Besides choosing a video and output location with the command line arguments, th
 
 - `--props`: React Props to pass to the root component of your video. Must be a serialized JSON string.
 - `--concurrency`: [How many CPU threads to use.](config#setconcurrency) Minimum 1. The maximum is the amount of threads you have.
-- `--png`: [Render a PNG sequence with transparency instead.](config#setoutputformat) If you use this option, the output location you pass must be a folder name.
+- `--png`: [Render a PNG sequence with transparency instead.](config#setoutputformat) If you use this option, the output location you pass must be a folder name. It will override `--format` flag.
+
+:::warning
+`--png` will be deprecated in thee upcoming major version release. Use `--format=png` instead of `--png` flag.
+:::
```

Comment:
Let's remove any mention of --png and add the information about PNG to the `--format` parameter. Ideally the user doesn't even know this parameter exists if it's deprecated

Reply:
okay done.